.github/workflows/pytest.yml

```diff
@@ -78,6 +78,30 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: '17'
+      - name: Install coursier
```

Comment:
Maybe use this instead?
https://github.com/marketplace/actions/coursier-setup-action

Or even a setup scala action

Reply:
I need is coursier, not a scala or sbt dependency.